.changeset/wet-apricots-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte-ux': patch
+---
+
+minor fix for Applayout
```

Comment:
Could you change this to "Update AppLayout nav to only show scrollbar when overflown (Windows, Mac with "Show scrollbars: always)"

Reply:
Pushed an updated changeset!

Reply:
Thanks.  You can actually edit the original file (`web-apricots-wait.md`).  Since you generated a new one, would you care to delete this file.

I need to have better docs for this process :(

Thanks!

Reply:
Docs would be nice in the future, whenever time allows :) Hopefully I've done it right this time.